dataprofiler/profilers/utils.py

```diff
@@ -483,3 +483,18 @@ def find_diff_of_dicts(dict1, dict2):
         diff = "unchanged"
 
     return diff
+
+def get_capacity(data, unit='M'):
+    """
+    Get size (capacity) of the input data
+
+    :param data: list or array of data
```

Comment:
needs unit and description of what unit options are avail

Reply:
added